bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -557,15 +557,11 @@ spec:
                 - name: RELATED_IMAGE_SYSTEM_MEMCACHED
                   value: mirror.gcr.io/library/memcached:1.5
                 - name: RELATED_IMAGE_BACKEND_REDIS
-                  value: quay.io/fedora/redis-6:latest
+                  value: quay.io/fedora/redis-7
```

Comment:
Why are we updating the redis image? Taught the flow was that by not updating to redis 7 we are forcing a move to external. Or is its just for development if so I don't think it should be in the CSV?

Reply:
ah is it for the preflights ?

Reply:
exactly, preflights, until we move to go libs we need to stick to these images :/